expo/expo
# [bare-expo] Fix pods

# Why

Fixes https://github.com/expo/expo/pull/10812#issuecomment-717114590.

# How

I removed `flipper_post_install`. Why? It causes all the problems with cocoapods. Moreover, we actually don't need this - we use swift and we need to add the `FB_SONARKIT_ENABLED` manually.


# Test Plan

- run `pod install` multiple times ✅
- check if flipper still works in the `bare-expo` ✅
